modules/custom/az_dam/config/quickstart/editor.editor.full_html.yml

```diff
@@ -0,0 +1,79 @@
+uuid: 0d90b549-88cc-42c1-a7f2-f813604f384f
```

Comment:
```suggestion
```

Reply:
Resolved locally. Will commit for review when all changes have been made.